bitnami/keycloak/templates/statefulset.yaml

```diff
@@ -294,23 +294,23 @@ spec:
           {{- if .Values.startupProbe.enabled }}
           startupProbe: {{- omit .Values.startupProbe "enabled" | toYaml | nindent 12 }}
             httpGet:
-              path: /auth/
```

Comment:
Has this path been changed in the new major? We are using it in our [cypress tests](    cy.visit('/auth/admin');) and we should modify them if so.